tests/bofire/strategies/botorch/test_base.py

```diff
@@ -727,110 +727,43 @@ def test_base_get_true_categorical_features():
+        (domains[3], SingleTaskGP),
+        (domains[4], ModelListGP),
     ],
 )
-def test_base_fit(domain, data, acquisition_function, expected):
-
-    myStrategy = DummyStrategy(domain=domain, acquisition_function=acquisition_function)
-    myStrategy.domain.set_experiments(data)
-
+def test_base_fit(domain, expected):
+    data = generate_experiments(domain, row_count=5, tol=1.0, force_all_categories=True)
+    domain.set_experiments(data)
+    acq = random.choice(list(AcquisitionFunctionEnum))
+    myStrategy = DummyStrategy(domain=domain, acquisition_function=acq)
     myStrategy.fit()
-
     assert isinstance(myStrategy.model, expected)
     domain.experiments = None
 
 
 @pytest.mark.parametrize(
-    "domain, data, acquisition_function",
+    "domain, acquisition_function",
     [
-        (
-            domains[0],
-            generate_experiments(
-                domains[0], row_count=5, tol=1.0, force_all_categories=True
-            ),
-            random.choice(list(AcquisitionFunctionEnum)),
-        ),
-        (
-            domains[1],
-            generate_experiments(
-                domains[1], row_count=5, tol=1.0, force_all_categories=True
-            ),
-            random.choice(list(AcquisitionFunctionEnum)),
-        ),
-        (
-            domains[2],
-            generate_experiments(
-                domains[2], row_count=5, tol=1.0, force_all_categories=True
-            ),
-            random.choice(list(AcquisitionFunctionEnum)),
-        ),
-        (
-            domains[3],
-            generate_experiments(
-                domains[3], row_count=5, tol=1.0, force_all_categories=True
-            ),
-            random.choice(list(AcquisitionFunctionEnum)),
-        ),
-        (
-            domains[4],
-            generate_experiments(
-                domains[4], row_count=5, tol=1.0, force_all_categories=True
-            ),
-            random.choice(list(AcquisitionFunctionEnum)),
-        ),
+        (domains[0], random.choice(list(AcquisitionFunctionEnum))),
+        (domains[1], random.choice(list(AcquisitionFunctionEnum))),
+        (domains[2], random.choice(list(AcquisitionFunctionEnum))),
+        (domains[3], random.choice(list(AcquisitionFunctionEnum))),
+        (domains[4], random.choice(list(AcquisitionFunctionEnum))),
     ],
 )
-def test_base_predict(domain, data, acquisition_function):
+def test_base_predict(domain, acquisition_function):
+    data = generate_experiments(domain, row_count=5, tol=1.0, force_all_categories=True)
     myStrategy = DummyStrategy(domain=domain, acquisition_function=acquisition_function)
 
     myStrategy.tell(data)
     predictions = myStrategy.predict(data)
 
-    assert len(predictions.columns.tolist()) == 2 * len(
-        domain.get_feature_keys(OutputFeature)
-    )
+    assert len(predictions.columns) == 2 * len(domain.get_feature_keys(OutputFeature))
     assert data.index[-1] == predictions.index[-1]
     domain.experiments = None
 
```

Comment:
Why do you changed the tests regarding base?

Reply:
No particular reason. I was at it because I had some failing test. You are right that the MR should be focused. I'll revert.

Reply:
No need for it, I was just wondering.